aegis_gym/envs/aegis_pusher.py

```diff
@@ -89,10 +95,10 @@ def __init__(
 
         self.episode_step = 0.0
         self.actions = th.zeros(self.num_actions, device=self.device)
-        self.last_actions = th.zeros(self.num_actions, device=self.device)
-        self.last_dof_vel = th.zeros(self.num_actions, device=self.device)
-        self.dof_pos = th.zeros(self.num_actions, device=self.device)
-        self.dof_vel = th.zeros(self.num_actions, device=self.device)
+        # self.last_actions = th.zeros(self.num_actions, device=self.device)
+        # self.last_dof_vel = th.zeros(self.num_actions, device=self.device)
+        self.dof_pos = th.zeros(6, device=self.device)
+        self.dof_vel = th.zeros(6, device=self.device)
```

Comment:
Will this work?

Reply:
Yes. Just like we currently have the TCP velocity commented out and unused, these additional past-action and joint-velocity observations are not needed for now. In the future, we may add an option to include such extra observations if required.

Reply:
So let's remove these commented lines, we can always add them back with git:
```suggestion
        self.dof_pos = th.zeros(6, device=self.device)
        self.dof_vel = th.zeros(6, device=self.device)
```